xpath-outcome-initiator/src/main/java/org/cristalise/kernel/persistency/outcomebuilder/field/StringField.java

```diff
@@ -374,6 +375,8 @@ public JSONObject getCommonFieldsNgDynamicForms() {
         //This can overwrite values set earlier, for example 'type' can be changed from INPUT to RATING
         readAppInfoDynamicForms(model, field);
 
+        field.put("cls", generateNgDynamicFormsCls());

```

Comment:
Changing the order might have significant side-effects. Why is this needed?

Reply:
I test it and looked into the order, it does not have any side-effects. It needs to run first readAppInfoDynamicForms(model, field) to set the value for ngcontainer.